test/integration/order.test.ts

```diff
@@ -77,4 +81,38 @@ describe('order', () => {
       owner: selfAlias,
     })
   })
+
+  it('creates an order transaction on chain', async () => {
+    demandCreateMock()
+    const { body: { id: orderId } } = await post(app, '/order', { parametersAttachmentId })
+
+    // submit to chain
+    const response = await post(app, `/order/${orderId}/creation`, {})
+    expect(response.status).to.equal(201)
+
+    const { id: transactionId, state } = response.body
+    expect(transactionId).to.match(
+      /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89ABab][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$/
+    )
+    expect(state).to.equal(TransactionState.submitted)
+
+    // check local transaction updates
+    const [transaction] = await db.getTransaction(transactionId)
+    expect(transaction.state).to.equal(TransactionState.finalised)
+    expect(transaction).to.contain({
+      state: 'finalised',
+      apiType: 'order',
+      transactionType: 'creation',
+    })
+
+    // check local capacity updates with token id
+    const [order] = await db.getDemand(orderId)
+    expect(order).to.contain({
+      id: orderId,
+      owner: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
```

Comment:
the convention is to return aliases for roles rather than addresses

Reply:
Updated

Reply:
still address?